lib/registries/addon/overview/-components/diff-manager/template.hbs

```diff
@@ -1,4 +1,4 @@
-{{did-update this.recalculateDiff @headRevisionId}}
+<div {{did-update this.recalculateDiff @headRevisionId}}></div>
```

Comment:
I know I proposed this but am kinda worried about setting a bad precedence. I wonder if there are better ways to do this.

Reply:
I'm not really sure. Nothing is immediately coming to mind. But I agree, we wouldn't want to encourage this pattern in the future.